submarine-cloud/hack/kind-cluster-build.sh

```diff
@@ -269,13 +269,16 @@ docker-image <image>' command to load images into nodes.
 EOF
 
 # Run submarine in kind cluster
+echo ""
```

Comment:
Is it must exists? Why?

Reply:
An empty line is output here to separate the following prompts.

Reply:
Thanks @liuxunorg , it's okay for me.